client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -239,10 +243,16 @@ public RssShuffleManager(SparkConf conf, boolean isDriver) {
     this.rssResubmitStage =
         rssConf.getBoolean(RssClientConfig.RSS_RESUBMIT_STAGE, false)
             && RssSparkShuffleUtils.isStageResubmitSupported();
+    this.taskBlockSendFailureRetry =
```

Comment:
this.taskBlockSendFailureRetryEnabled = rssConf.getBoolean(RssClientConf.RSS_TASK_FAILURE_RETRY_ENABLED);